src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -549,6 +618,128 @@ std::set<std::vector<element::Type>> jit_greater_equal_emitter::get_supported_pr
+        OPENVINO_THROW("Can't create jit eltwise kernel for LESS");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_less_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
+                               const std::vector<size_t>& out_vec_idxs) const {
+    OPENVINO_ASSERT(exec_prc_ == element::f32, "JIT Less emitter supports only f32 precision");
+    
+    VReg src0 = VReg(in_vec_idxs[0]);
+    VReg src1 = VReg(in_vec_idxs[1]); 
+    VReg dst = VReg(out_vec_idxs[0]);
+    FReg one = FReg(aux_fp_gpr_idxs[0]);
+
+    h->vmv_v_x(dst, x0);
+
+    load_table_val("one", one);
+
+    h->vmflt_vv(mask_vreg(), src0, src1);
+    
+    h->vfadd_vf(dst, dst, one, VM::masked);
+}
+
+void jit_less_emitter::register_table_entries() {
+    push_arg_entry_of("one", CONST_1_F);
+}
+std::set<std::vector<element::Type>> jit_less_emitter::get_supported_precisions(
+    const std::shared_ptr<ov::Node>& node) {
+    return {{element::f32, element::f32}};
+}
+
+/// LOGICAL OR ///
+jit_logical_or_emitter::jit_logical_or_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+}
+
+jit_logical_or_emitter::jit_logical_or_emitter(jit_generator* host, cpu_isa_t host_isa,const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+}
+
+size_t jit_logical_or_emitter::get_inputs_num() const {
+    return 2;
+}
+size_t jit_logical_or_emitter::aux_vecs_count() const {
+    return 1;  
+}
+size_t jit_logical_or_emitter::aux_fp_gprs_count() const {
+    return 2;
+}
+
+void jit_logical_or_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs,
+                                      const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == ov::intel_cpu::riscv64::cpu_isa_t::gv) {
+        emit_isa<ov::intel_cpu::riscv64::cpu_isa_t::gv>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_logical_or_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
+                                     const std::vector<size_t>& out_vec_idxs) const {
+    OPENVINO_ASSERT(exec_prc_ == element::f32, "JIT Logical OR emitter supports only f32 precision");
+    
+
+   
+    const VReg src0 = VReg(in_vec_idxs[0]);
+    const VReg src1 = VReg(in_vec_idxs[1]);
+    const VReg dst = VReg(out_vec_idxs[0]);
+    const VReg tmp = VReg(aux_vec_idxs[0]);
+    FReg fzero = FReg(aux_fp_gpr_idxs[0]);
+    FReg fone = FReg(aux_fp_gpr_idxs[1]);
+
+    h->fmv_w_x(fzero, x0);
+    load_table_val("one", fone);
+
+    h->vmv_v_x(dst, x0);
+
+    h->vmfne_vf(mask_vreg(), src0, fzero);  
+    h->vfadd_vf(dst, dst, fone, VM::masked); 
+
+    h->vmfne_vf(mask_vreg(), src1, fzero);  
+    h->vmseq_vx(tmp, dst, x0);              
+    h->vand_vv(mask_vreg(), mask_vreg(), tmp); 
+    h->vfadd_vf(dst, dst, fone, VM::masked); 
```

Comment:
I believe that we can improve this implementation a little bit more - the impl might have less instructions :)
The idea is in your two lines which handle `src0`:
```
h->vmfne_vf(mask_vreg(), src0, fzero);  
h->vfadd_vf(dst, dst, fone, VM::masked); 
```
What's about to use `xor` instruction instead of `vfadd` - since the operation is `LogicalOr`? 😄 
And then we can call the same logic for `src1`.

What do you think? (You just need to use another register instead of `FReg fone`).